### DISCUSSION TITLE
直链预览图片

### DISCUSSION THREAD

**Discussion by Ylanw** [spencerwooo/onedrive-vercel-index#806] [Ideas]
能不能直接通过直链预览图片，不下载到本地

**Comment by hirockstar** [comment]
there is two method for that 
https://drive.swo.moe/api/raw/?path=/%F0%9F%8D%87%20Wallpaper/chasm_surface.png
https://drive.swo.moe/api/thumbnail/?path=/%F0%9F%8D%87%20Wallpaper/chasm_surface.png
